packages/net-stubbing/lib/server/util.ts

```diff
@@ -220,11 +220,19 @@ export function getBodyStream (body: Buffer | string | Readable | undefined, opt
     return writable.end()
   }
 
-  delay ? setTimeout(sendBody, delay) : sendBody()
+  delay ? await wait(sendBody, delay) : sendBody()
 
   return pt
 }
 
+function wait (fn, ms) {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(fn())
+    }, ms)
+  })
+}
+
```

Comment:
What is the right place for this utility function?

Reply:
Bluebird offers `Promise.delay` which may meet your needs. If it's just used in this function/file then this is fine.

Reply:
I changed the code with `Promise.delay`.

Reply:
@sainthkh There seems to be a type error with the change